src/main/java/core/basesyntax/behaviour/Suppliable.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.behaviour;
+
+public interface Suppliable {
+    Object getRandom();
```

Comment:
Change the return type of getRandom() method from 'Object' to a more specific type related to figures (e.g., 'Figure').